lib/puppeteer/frame_manager.rb

```diff
@@ -65,8 +65,17 @@ def initialize(client, page, ignore_https_errors, timeout_settings)
     )
     frame_tree = results.last['frameTree']
     handle_frame_tree(frame_tree)
+    downloads_folder = ENV.fetch 'PUPPETEER_DOWNLOADS_FOLDER', '/tmp'
     await_all(
-      @client.async_send_message('Page.setLifecycleEventsEnabled', enabled: true),
+      @client.async_send_message(
+        'Page.setLifecycleEventsEnabled',
+        enabled: true
+      ),
+      @client.async_send_message(
+        'Page.setDownloadBehavior',
+        behavior: 'allow',
```

Comment:
https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-setDownloadBehavior

This method seems deprecated.
Does this work?